### PR TITLE
permission query editor behind isBeta flag

### PR DESCRIPTION
**Pull request by sorenbs**

### CODE REVIEW

File: src/views/PermissionsView/PermissionPopup/PermissionConditions.tsx [hunk 1]
```diff
@@ -79,8 +82,44 @@ export default (props) => (
...
+                Rule
+              </div>
+            </ConditionButton>
+          </div>}
         </div>
       </div>
+
+      {props.rule === 'GRAPH' && <div
+        className={cx($p.mt25)}>
+        <QueryEditor className={cx($p.z9999)} schema={buildClientSchema(JSON.parse(props.permissionSchema))}
```
Comment (schickling) [inline src/views/PermissionsView/PermissionPopup/PermissionConditions.tsx]: Please spread out props over new lines

File: src/views/PermissionsView/PermissionPopup/PermissionPopup.tsx [hunk 2]
```diff
@@ -145,7 +161,10 @@ class PermissionPopup extends React.Component<Props, State> {
               />
             )}
             {selectedOperation !== null && (
-              <PermissionConditions userType={userType} setUserType={this.setUserType} />
+              <PermissionConditions userType={userType} isBetaCustomer={this.props.isBetaCustomer} ruleType={rule}
```
Comment (schickling) [inline src/views/PermissionsView/PermissionPopup/PermissionPopup.tsx]: Please spread out props over new lines

File: src/views/PermissionsView/PermissionPopup/PermissionPopup.tsx [hunk 3]
```diff
@@ -245,9 +272,17 @@ export const AddPermissionPopup = Relay.createContainer(withRouter(MappedPermiss
   fragments: {
     viewer: () => Relay.QL`
       fragment on Viewer {
+        user {
+          crm{
```
Comment (schickling) [inline src/views/PermissionsView/PermissionPopup/PermissionPopup.tsx]: Spaces are missing

File: src/views/PermissionsView/PermissionPopup/PermissionPopup.tsx [hunk 4]
```diff
@@ -214,6 +234,11 @@ class PermissionPopup extends React.Component<Props, State> {
 const MappedPermissionPopup = mapProps({
   permission: props => props.node || null,
   model: props => (props.viewer && props.viewer.model) || props.node.model,
+  isBetaCustomer: props => (props.viewer &&
+  props.viewer.user &&
```
Comment (schickling) [inline src/views/PermissionsView/PermissionPopup/PermissionPopup.tsx]: Can there be a case where there is now user, crm & information?

File: src/styles/codemirror.css [hunk 5]
```diff
@@ -1,4 +1,8 @@
 .code-generation-popup .CodeMirror {
```
Comment (schickling) [inline src/styles/codemirror.css]: @timsuchanek was this file used before?

File: src/views/PermissionsView/PermissionPopup/PermissionConditions.tsx (hunk 1 above)
Comment (sorenbs) [inline src/views/PermissionsView/PermissionPopup/PermissionConditions.tsx]: Id this a general rule - ie, should this

```
<OperationChooser selectedOperation={selectedOperation} setOperation={this.setOperation}/>
```
be
```
<OperationChooser
  selectedOperation={selectedOperation}
  setOperation={this.setOperation}/>
```
?

File: src/styles/codemirror.css (hunk 5 above)
Comment (sorenbs) [inline src/styles/codemirror.css]: It's used by the code generater

**Pull request merged**